Open_Source_Events/GSSOC/README.md

```diff
@@ -0,0 +1,6 @@
+# GirlScript Summer Of Code
```

Comment:
Add a center align tag so that # GirlScript Summer Of Code is in center.

Add an h2  `## About GSSoC` heading after that